common/src/main/java/org/cloudfoundry/autosleep/config/Config.java

```diff
@@ -113,6 +113,12 @@
         String SECRET = "secret";
 
     }
+    
+    interface OrgEnrollmentParameters {
+        enum EnrolledState {
+            backoffice_enrolled, backoffice_opted_out
```

Comment:
Good point. Missed that. But a question again here: This table anyways captures the configuration for orgs via back office APIs which can be either enrolled or opted-out. How different is auto-enrolled different from backoffice-enrolled?